### PR TITLE
Create components for calculator UI

### PR DESCRIPTION
**Pull request by Bellagirl-maker**
-Create a React component that will hold the core functionality: a calculator

### CODE REVIEW

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Isabella, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Great job creating the calculator.

## Required Changes ♻️
## I'm leaving some requirements here in case there are no changes made in the actual file.
- [ ] good job using the readme template from Microversers, however, try to be a little more specific in the "install" and usage part. describe it in such a way that a person not related to the tech industry (reqruiter) can install and run the project.
- [ ] there are still some files created by the CRA which need to be removed for example in the Public folder
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bellagirl-maker** [comment]
@mikethreels Done with the requested changes

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️

- [ ] As mentioned by the previous reviewer,  there are still some files created by the CRA which need to be removed for example in the Public folder, you still have the logo, robots.txt and favicon which should be deleted. Kindly make the change 👍🏽 

Check the comments under the review.
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bellagirl-maker** [comment]
@vickymarz should I delete only the three files you mentioned or all files in the public folder?

**Comment by Bellagirl-maker** [comment]
@vickymarz I have removed the files

**Comment by victorokoroji** [comment]
@Bellagirl-maker only the files i mentioned please. Other files are important for the app to work

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
```
Comment (youmari) [inline src/index.js]: - [  ]OPTIONAL  still has some files to be removed like setup, report, and app.test.js 
Screenshot 2023-05-02 at 22 39 18

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @Bellagirl-maker 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**